scripts/github-last-contribution.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import argparse
+import urllib.request
+from bs4 import BeautifulSoup
+'''
+For using this script call the get_usernames() function and pass all the
+usernames in a space separated string.
+'''
+
+
+def last_contribution(user_name):
+    profile_url = "https://github.com/{}/zulip".format(user_name)
+    try:
+        profile_page = BeautifulSoup(
+            urllib.request.urlopen(profile_url), "lxml")
+    except urllib.error.URLError:
+        return "Error"
+    time_tag = profile_page.findAll('relative-time', datetime=True)
+    # I am retrieving the datetime data below and splitting it into date and
+    # time.
+    time_stamp = time_tag[0].get("datetime")[:-1].split("T")
+    name_and_time_stamp = [user_name] + time_stamp
+    return name_and_time_stamp
+
+
+def get_usernames(user_names):
+    user_names = user_names.split(' ')
+    print("Here are the timestamps for the last commit to the Zulip forks:")
+    for user_name in user_names:
+        name_and_time_stamp = last_contribution(user_name)
+        if name_and_time_stamp == "Error":
```

Comment:
I think its better to raise an error in above function and just catch it here. Returning 'Eror' does not look like good practice